flag.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2015 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package charmrepo
+
+import (
+	"flag"
+	"os"
+
+	jujutesting "github.com/juju/testing"
+)
+
+var noTestMongoJs *bool = flag.Bool("notest-mongojs", false, "Disable MongoDB tests that require JavaScript")
```

Comment:
What's this doing in a non-test file? It seems entirely test related to me and AFAICS there's no non-test code that uses it.